docs/fomega/evaluation/CEK.md

```diff
@@ -17,10 +17,11 @@ The relevant files are:
 
 Version 1:
 
-https://github.com/jmchapman/plutus/blob/master/metatheory/Algorithmic/CEKV.lagda.md
+https://github.com/input-output-hk/plutus/blob/master/metatheory/Algorithmic/CEKV.lagda.md
```

Comment:
For things like this I think relative links are often better.

Reply:
done.